packages/zowe-explorer/src/shared/refresh.ts

```diff
@@ -26,6 +29,8 @@ import * as globals from "../globals";
  * @param {IZoweTree} treeProvider
  */
 export async function refreshAll(treeProvider: IZoweTree<IZoweTreeNode>) {
+    await readConfigFromDisk();
+    await Profiles.getInstance().refresh(ZoweExplorerApiRegister.getInstance());
```

Comment:
This makes the extension work seamlessly with the CLI
Whether that's doing
- ZE: Log in to Authentication Service  -> CLI commands
- CLI auth login OR CLI config auto-init -> ZE browsing (after a soft refresh using the button)